forge/forge/op/loss.py

```diff
@@ -119,3 +119,18 @@ def forward(self, predictions, labels):
         square = Multiply("square", diff, diff)
         loss = reduce_loss(self.reduction, square)
         return loss
+
+
+class NLLLoss(ForgeModule):
+    def __init__(self, name: str):
```

Comment:
Add comment as in CrossEntropy as well, but with note that input should be in the log-space